### PR TITLE
Issue credential

### PR DESCRIPTION
**Pull request by amanji**
Adds Issue Credential protocol for Issuer.

Depends on https://github.com/bcgov/aries-vcr/pull/658

### CODE REVIEW

File: controller/src/utils/issuer-registration.ts [hunk 1]
```diff
@@ -165,33 +166,55 @@ function formatAttributeMappings(
   attributes: string[],
   metadata: CredentialMetadata
 ): ModelMapping[] {
+
   const mappings = [] as ModelMapping[];
 
-  const standardAttributes = attributes.filter((attribute) => {
+  const baseAtributeMapping = (attribute: string) => ({
```
Comment (esune) [inline controller/src/utils/issuer-registration.ts:172]: Nit.: typo `atTribute`

**Review by esune — COMMENTED** [review]
Looking good so far

File: controller/src/models/errors.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ export class UndefinedAppError extends Error {}
 export class DuplicatedProfileError extends Error {}
 
 export class AriesAgentError extends FeathersError {
-  constructor(message: string | Error, code: number | undefined, data?: any) {
+  constructor(message: string | Error, code: number | undefined, data?: unknown) {
```
Comment (esune) [inline controller/src/models/errors.ts:8]: Was this throwing new  compilation errors?

File: docker/manage [hunk 3]
```diff
@@ -189,6 +189,12 @@ case "$COMMAND" in
     configureEnvironment $@
     build ${_startupParams}
     ;;
+  test)
+    _startupParams=$(getStartupParams $@)
+    configureEnvironment "$@"
+	  build
```
Comment (esune) [inline docker/manage:195]: You may want to use `build ${_startupParams}` as in line `190` for consistency, unless there is a specific reason not to.

File: controller/src/models/errors.ts (hunk 2 above)
Comment (amanji) [reply controller/src/models/errors.ts:8]: No the linter was complaining. The superclass uses `unknown` for the type.

File: docker/manage (hunk 3 above)
Comment (amanji) [reply docker/manage:195]: This was committed accidentally. Will remove

**Pull request closed without merging**